### DISCUSSION TITLE
ISO Mount/Dismount Support

### DISCUSSION THREAD

**Discussion by RyanCVE** [PSAppDeployToolkit/PSAppDeployToolkit#1098] [Ideas]
I have a lot of large vendor-supplied applications which come as .iso files, it would be a nice feature if I was able to mount and dismount these in the same manner as the Wim support.

Would it be possible to extend support for this format also?

**Comment by mjr4077au** [comment]
Hi @RyanCVE, can you please raise this as a proper feature request under the Issues section of our project so we can track it better?
